src/matlab2tikz.m

```diff
@@ -1791,6 +1791,23 @@ function assertRegularAxes(handle)
         % plot them
         strPart = cell(1, length(dataCell));
         for k = 1:length(dataCell)
+            %If drawOptions has error bar style options, then print these
+            %options on the variable errorBarOpts and remove them from
+            %drawOptions
+            if (opts_has(drawOptions,'error bar style'))
+                tmpOptions = opts_new();
```

Comment:
Since we have the opportunity to standardise the code, we can initialize the error bar options here:
```
tmpOptions = opts_add(tmpOptions,'error bars/.cd','');
tmpOptions = opts_add(tmpOptions,'y dir','both');
tmpOptions = opts_add(tmpOptions,'y explicit','');
```